### ISSUE TITLE
Login method is invoked twice in channels.js file

### ISSUE THREAD

**Issue by rondaniel00** [feathersjs/authentication#673]
Login method is invoked twice in channels.js file.
app.on('login', (authResult, { connection }) => { // CALLED TWICE  } ));

Verified that  client.authenticate() is called only once.

**Comment by marshallswain** [comment]
@rondaniel00 I believe you mentioned that there was something specific to your app with this issue?  Is that for sure the case?

**Comment by daffl** [comment]
I can confirm that this seems to be happening everywhere. Try the [feathers-chat](https://github.com/feathersjs/feathers-chat) logging the `login` event. When refreshing the chat page (once authenticated) it will show up twice.

**Comment by 8BallBomBom** [comment]
Also experiencing this.
Thought i had something acting up on my client.

**Comment by koraniar** [comment]
I have the same problem and I verified that client.authenticate() is called only once

**Comment by ricardopolo** [comment]
I think this may be related, that the feathers cliente default behavior stabilsih the conecction using long pooling and the upgrade to websockets...

@koraniar would you test modifying the feathers client parameters to see how if is that the root cause?

**Comment by daffl** [comment]
Duplicate of https://github.com/feathersjs/feathers/issues/1174, will be tracked there.
